spec/requests/admin/administrators/administrators_spec.rb

```diff
@@ -167,6 +167,16 @@
       expect(admin_profile.discarded?).to be false
       expect(admin_user.discarded?).to be false
     end
+
+    describe "when an audited action", versioning: true do
```

Comment:
I added a few of these tests for sanity checking and mostly to confirm whodunnit is being set to the current user.